app/models/form/question.rb

```diff
@@ -317,7 +317,7 @@ def evaluate_condition(condition, log)
   end
 
   def enabled_inferred_answers(inferred_answers, log)
-    inferred_answers.filter { |_key, value| value.all? { |condition_key, condition_value| log[condition_key] == condition_value } }
+    inferred_answers.filter { |_key, value| value.all? { |condition_key, condition_value| log.public_send(condition_key) == condition_value } }
```

Comment:
needed so the inferred_answers can acccept log methods as well as attributes

Reply:
Would it work if we delegated the location name to the log? Wouldn't need to change any methods

Reply:
I don't think so - it's the :scheme_has_multilpe_locations? method that I need the public send for

Reply:
I feel pretty strongly that we shouldn't have nested one line code blocks, I think this should be spread across multiple lines.

I also don't know how these inferred answers things work, their form or what they're for exactly, so it would be good to rename `|_key, value|` with something a little more descriptive.

Reply:
I think these are good ideas